src/coreComponents/constitutive/solid/LinearElasticTransverseIsotropic.cpp

```diff
@@ -109,6 +117,12 @@ void LinearElasticTransverseIsotropic::PostProcessInput()
   real64 const c44Default = Gat;
   real64 const c66Default = 0.5 * Et / ( 1 + Nut );
 
+  //wu40: Use arithmetic average for bulk and shear moduli, which are further used in surface generator for nodal force calculation.
+  real64 arithmeticPoissonRatio = (m_defaultPoissonTransverse + m_defaultPoissonAxialTransverse) / 2;
+  real64 arithmeticYoungsModulus = (m_defaultYoungsModulusTransverse + m_defaultYoungsModulusAxial + 2 * m_defaultShearModulusAxialTransverse * (1 + arithmeticPoissonRatio)) / 3;
+
+  real64 const averageBulkModulus = arithmeticYoungsModulus / 3 / (1 - 2 * arithmeticPoissonRatio);
+  real64 const averageShearModulus = arithmeticYoungsModulus / 2 / (1 + arithmeticPoissonRatio);
```

Comment:
This will mean that only the default value will be used. If a heterogeneous property field is specified, this will not catch that. We have to relate the isotropic moduli (E, nu, K, G) with the anisotropic parameters in each element.